src/pcp/atop/showprocs.c

```diff
@@ -46,6 +46,10 @@ static char     *columnhead[] = {
 	[MSORTGPU]= "GPU",
 };
 
+static count_t sys_btime = 0;
```

Comment:
Let's declare a 'boottime' variable in atop.h with the other system-wide variables there (eg pagesize, etc).